precise/scripts/train.py

```diff
@@ -104,9 +106,13 @@ def on_epoch_end(_a, _b):
             self.hash_to_ind = {}
 
         self.callbacks = [
-            checkpoint, TensorBoard(
-                log_dir=self.model_base + '.logs',
-            ), LambdaCallback(on_epoch_end=on_epoch_end)
+          checkpoint,
+          #  TensorBoard( # Disables tensorboard due to compat with tf2
```

Comment:
Here

Reply:
Addressed this is latest commit 'General cleanup for merge'.